format/index.md

```diff
@@ -541,8 +536,8 @@ array of values, whereas a `self` link does not).
 A JSON API document **MAY** include information about its implementation under
 a top level `jsonapi` member. If present, it **MUST** be an object containing
 a `version` member whose value is a string indicating the highest JSON API
-version supported. Other than `meta`, servers **MUST NOT** include any
-additional members.
+version supported. Other than `meta`, which **MUST** contain a [meta] object if
+included, servers **MUST NOT** include any additional members.
```

Comment:
this is redundant with this line in the document structure section:
> Unless otherwise noted, objects defined by this specification MUST NOT contain any additional members.